.github/actions/slack-file/action.yml

```diff
@@ -0,0 +1,20 @@
+name: "Send Message"
+description: |
+  Sends the contents of a file as a slack message.
+inputs:
+  payload-file:
+    description: Message file.  If the file does not exist, silently proceed without sending a message.
+    required: false
+  webhook:
+    description: Webhook for the message.
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        build_url="${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
```

Comment:
turns out this as specific as we can be with the UI, we can point to the workflow, but not the job.